packages/cubejs-schema-compiler/src/adapter/PreAggregations.js

```diff
@@ -452,13 +546,24 @@ export class PreAggregations {
   }
 
   // eslint-disable-next-line no-unused-vars
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   getCubeLattice(cube, preAggregationName, preAggregation) {
     throw new UserError('Auto rollups supported only in Enterprise version');
   }
 
+  /**
+   * Returns pre-agg which determined as applicable for the query (the first one
+   * from the list of potencially applicable pre-aggs). The order of the incoming
+   * list of potencially applicable pre-aggs is equal to the order of these pre-
```

Comment:
Nit: "The order of the potentially applicable pre-aggs is the same as the order in which these pre-aggs appear in the schema file"

Generally speaking, I'm trying to help with the English formulation here (heh, not native Engslish speaker myself either). If this feels too nit-picky, I'm happy to stop :)